student.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './person'
+class Student < Person
+  def initialize(age, classroom, parent_permission, name = 'Unknown')
```

Comment:
- [ ] I noticed that the previous reviewer mentioned to update the default value of the `parent_permission` parameter to be `true`(as a boolean) but instead of that, you remove the value. Make sure you update it again by assigning it to its expected value(i.e. `parent_permission: true`)